purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -171,30 +173,19 @@ class PostingTransactionsTests {
             onError = { _, _ -> }
         )
         assertThat(postedProductInfoSlot.isCaptured).isTrue
-        assertThat(postedProductInfoSlot.captured.duration).isEqualTo(expectedSubscriptionPeriod)
-        assertThat(postedProductInfoSlot.captured.introDuration).isEqualTo(expectedIntroPricePeriod)
-        assertThat(postedProductInfoSlot.captured.trialDuration).isEqualTo(expectedFreeTrialPeriod)
-        verify(exactly = 1) {
-            backendMock.postReceiptData(
-                purchaseToken = any(),
-                appUserID = appUserId,
-                isRestore = any(),
-                observerMode = any(),
-                subscriberAttributes = expectedAttributes.toBackendMap(),
-                storeAppUserID = any(),
-                receiptInfo = any(),
-                marketplace = any(),
-                onSuccess = any(),
-                onError = any()
-            )
-        }
+        assertThat(postedProductInfoSlot.captured.duration).isEqualTo(mockStoreProduct.subscriptionPeriod)
     }
 
     @Test
-    fun `inapps send null durations when posting to backend`() {
+    fun `purchaseOptionId is sent when posting to backend`() {
```

Comment:
Should we verify the pricing phases are correct in another test as well? We can add that in another PR but I would add a TODO in that case so we don't forget.

Reply:
Yes definitely!  I have another branch started w pricing phase validation, in the BackendTests